test/dummy_test.py

```diff
@@ -0,0 +1,6 @@
+from unittest import TestCase
+
+
+class TestDummy(TestCase):
```

Comment:
are we getting unit tests too now?

Reply:
just a place holder for future unit tests